client/layouts/blog.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import Head from 'next/head'
+import { parseISO, format } from 'date-fns'
+import {
+    useColorMode,
+    Heading,
+    Text,
+    Flex,
+    Stack,
+    Avatar
+} from '@chakra-ui/react'
+import { useRouter } from 'next/router'
+
+import Container from '../components/Container'
+
+export default function BlogLayout({ children, frontMatter }) {
+    const { colorMode } = useColorMode()
+    const textColor = {
+        light: 'gray.700',
+        dark: 'gray.400'
+    }
+    const router = useRouter()
+    const slug = router.asPath.replace('/blog', '')
```

Comment:
Why do we need `router` and `slug` defined here? They don't seem to be used anywhere